src/Dangl.SchneidControl/Data/EmailType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Dangl.SchneidControl.Data
+{
+    public enum EmailType
+    {
+        Information = 0,
+
+        Warning = 1,
+
+        Error = 2,
+    }
```

Comment:
I was here thinking more of message types, like ´LowTemperatureWarning`, which we send if we detect that the heating system has failed. So in the future we can add other types.

Then, when we check if we should send a low temperature warning, we can check if there's been one sent in e.g. the last 12 hours.